README.md

```diff
@@ -1,7 +1,8 @@
 ---
-title: README
+title: Open Governance
```

Comment:
This won't get overriden by Obsidian Linter will it?

Reply:
https://github.com/redbrick/open-governance/blob/d6c52707abbc14a8a69f11746a4ac5ff4f0dcc01/.obsidian/plugins/obsidian-linter/data.json#L57

Appears like it will, unless I'm reading wrong

Reply:
Yeah, linter probably will override. I don't think a linter for the title is necessary, if we can provide a template with frontmatter fields and reasonable defaults that new entries use.